Tests/pens/ttGlyphPen_test.py

```diff
@@ -290,6 +290,310 @@ def test_scaled_component_bounds(self):
         self.assertGlyphBoundsEqual(uni0302_uni0300, (-278, 745, 148, 1025))
 
 
+class TTGlyphPointPenTest:
+    def runEndToEnd(self, filename):
+        font = ttLib.TTFont()
+        ttx_path = os.path.join(
+            os.path.abspath(os.path.dirname(os.path.realpath(__file__))),
+            "..",
+            "ttLib",
+            "data",
+            filename,
+        )
+        font.importXML(ttx_path)
+
+        glyphSet = font.getGlyphSet()
+        glyfTable = font["glyf"]
+        pen = TTGlyphPointPen(font.getGlyphSet())
+
+        for name in font.getGlyphOrder():
+            oldGlyph = glyphSet[name]
+            oldGlyph.drawPoints(pen)
+            oldGlyph = oldGlyph._glyph
+            newGlyph = pen.glyph()
+
+            if hasattr(oldGlyph, "program"):
+                newGlyph.program = oldGlyph.program
+
+            assert oldGlyph.compile(glyfTable) == newGlyph.compile(glyfTable)
+
+    def test_e2e_linesAndSimpleComponents(self):
+        self.runEndToEnd("TestTTF-Regular.ttx")
+
+    def test_e2e_curvesAndComponentTransforms(self):
+        self.runEndToEnd("TestTTFComplex-Regular.ttx")
+
+    def test_glyph_simple(self):
+        pen = TTGlyphPointPen(None)
+        pen.beginPath()
+        pen.addPoint((50, 0), "line")
+        pen.addPoint((450, 0), "line")
+        pen.addPoint((450, 700), "line")
+        pen.addPoint((50, 700), "line")
+        pen.endPath()
+        glyph = pen.glyph()
+        assert glyph.numberOfContours == 1
+        assert glyph.endPtsOfContours == [3]
+
+    def test_glyph_errorOnCurve(self):
+        pen = TTGlyphPointPen(None)
+        pen.beginPath()
+        with pytest.raises(NotImplementedError):
+            pen.addPoint((0, 0), "curve")
+
+    def test_beginPath_beginPathOnOpenPath(self):
+        pen = TTGlyphPointPen(None)
+        pen.beginPath()
+        pen.addPoint((0, 0))
+        with pytest.raises(AssertionError):
+            pen.beginPath()
+
+    def test_closePath_ignoresAnchors(self):
```

Comment:
what do you mean here by "anchors"? I only see a beginPath and an endPath (an empty contour)